src/commands/base.ts

```diff
@@ -242,6 +243,7 @@ export default abstract class Base extends Command {
             || this.userConfig?.project
             || ''
         this.orgId = flags['org'] || this.repoConfig?.org?.id || this.userConfig?.org?.id
+        this.organization_name = this.repoConfig?.org?.display_name || this.userConfig?.org?.display_name || ''
```

Comment:
you could make this a little more generic, so that we can access any properties
```suggestion
        this.organization = this.repoConfig?.org || this.userConfig?.org
        this.orgId = flags['org'] || this.organization?.id
```